pkg/cmd/server/crypto/crypto.go

```diff
@@ -174,12 +174,9 @@ func GetTLSCertificateConfig(certFile, keyFile string) (*TLSCertificateConfig, e
 	return &TLSCertificateConfig{certs, key}, nil
 }
 
-var (
-	// Default ca certs to be long-lived
-	caLifetime = time.Hour * 24 * 365 * 5
-
-	// Default templates to last for two years
-	lifetime = time.Hour * 24 * 365 * 2
+const (
+	DefaultCertificateLifetimeInDays   = 365 * 2 // 2 years
```

Comment:
I'm using them as default values in different places. Do you suggest to copy&paste this value?